pcsx2/gui/Panels/GSWindowPanel.cpp

```diff
@@ -140,16 +196,21 @@ void Panels::GSWindowSettingsPanel::ApplyConfigToGui( AppConfig& configToApply,
 
 	if( !(flags & AppConfig::APPLY_FLAG_FROM_PRESET) )	
 	{//Presets don't control these values
-		m_check_CloseGS		->SetValue( conf.CloseOnEsc );
-		m_check_Fullscreen	->SetValue( conf.DefaultToFullscreen );
-		m_check_HideMouse	->SetValue( conf.AlwaysHideMouse );
-		m_check_SizeLock	->SetValue( conf.DisableResizeBorders );
-
-		m_combo_AspectRatio	->SetSelection( (int)conf.AspectRatio );
-		m_text_Zoom			->ChangeValue( conf.Zoom.ToString() );
+		m_check_CloseGS					->SetValue( conf.CloseOnEsc );
+		m_check_Fullscreen				->SetValue( conf.DefaultToFullscreen );
+		m_check_HideMouse				->SetValue( conf.AlwaysHideMouse );
+		m_check_ScalingInterlaced		->SetValue( conf.DisableScalingInterlaceCompensation);
+		m_check_ScalingInterlaced		->Enable( (conf.AspectRatio != AspectRatio_Stretch && conf.ScalingType != ScalingType_Fit) || conf.AspectRatio == AspectRatio_Frame );
+		m_check_SizeLock				->SetValue( conf.DisableResizeBorders );
+
+		m_combo_AspectRatio				->SetSelection( (int)conf.AspectRatio );
+		m_text_Zoom						->ChangeValue( conf.Zoom.ToString() );
+		m_combo_ScalingType				->SetSelection( (int)conf.ScalingType );
+		m_combo_ScalingType				->Enable( conf.AspectRatio != AspectRatio_Stretch );
+		m_combo_FMVAspectRatioSwitch	->SetSelection( (int)conf.FMVAspectRatioSwitch );
+		m_combo_FMVAspectRatioSwitch	->Enable( conf.AspectRatio != AspectRatio_4_3 );
```

Comment:
Doing all the grayouts only on applying the settings isn't enough, you also need to bind them to event handlers to make the grayouts immediate on customization of the combo box.